### DISCUSSION TITLE
why isn't entangle using `$`

### DISCUSSION THREAD

**Discussion by andy-UKC** [rawilk/filament-quill#58] [General]
New to Livewire and filament etc, so forgive my ignorance, but why in the quill-editor.blade.php file is the `\$` ommitted from the entangle if it is not disabled?

disabled section, with `\$`: https://github.com/rawilk/filament-quill/blob/73ed558e919d82d816e69d4f5923d43089f9f12d/resources/views/quill-editor.blade.php#L47

active section, without `\$`: https://github.com/rawilk/filament-quill/blob/73ed558e919d82d816e69d4f5923d43089f9f12d/resources/views/quill-editor.blade.php#L68

the error I'm getting is fixed if you add the `\$` to the latter. When you add a debounce or `onBlur:true` to `->live()`, you get an error like:

```
Alpine Expression Error: Unexpected number '.500'. Expected '}' to end an object literal.

Expression: "quill({ state: $wire.entangle('data.description').live.debounce.500,
```

**Comment by rawilk** [comment]
That is just a typo that I haven't caught. I'll be sure to fix that in the next release I'm working on right now.
